src/DropDownMenu/DropDownMenu.js

```diff
@@ -241,7 +261,7 @@ class DropDownMenu extends Component {
 
   handleTouchTapControl = (event) => {
     event.preventDefault();
-    if (!this.props.disabled) {
+    if (!this.props.disabled && !(this.props.multiple && this.state.open)) {
```

Comment:
You are right. The ClearFix component is covered by the Popover so no touch events are possible.